Wp 525 enable video for webrtc

pc-m opened the pull request.

pc-m committed:
endpoint: add the "add --template" subcommand

this can be used to add a template to a SIP endpoint

Usage:
  endpoint sip add --template <template-uuid> <endpoint_uuid>

pc-m committed:
contribs: add a script to enable video on webrtc endpoints

pc-m committed:
packaging: install the examples on the stack

sduthil commented on contribs/examples/enabled-video.sh:
```diff
@@ -0,0 +1,91 @@
+    esac
+done
+
+if [[ ! -v TENANT_UUID ]]; then
```
This condition is also true if `$TENANT_UUID=''`. So I think `[ -z "$TENANT_UUID" ]` would be better.

sduthil commented on contribs/examples/enabled-video.sh:
```diff
@@ -0,0 +1,91 @@
+}
+
+# Check for dependencies
+if ! command -v wazo-auth-cli &> /dev/null; then
```
Nice! I did not know about the `command -v` trick! Thank you!

sduthil commented on contribs/examples/enabled-video.sh:
```diff
@@ -0,0 +1,91 @@
+
+# Get a TOKEN
+if [[ -v AUTH_USERNAME ]]; then
+    echo -n "Password: "
```
```suggestion
    echo -n "Password for $AUTH_USERNAME: "
```

sduthil commented on contribs/examples/enabled-video.sh:
```diff
@@ -0,0 +1,91 @@
+fi
+
+# Fetch all WebRTC video endpoints
+readarray -t WEBRTC_VIDEO_ENDPOINT_UUIDS < <(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_VIDEO_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value | awk -F ' ' '{ print $1 }')
```
```suggestion
readarray -t WEBRTC_VIDEO_ENDPOINT_UUIDS < <(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_VIDEO_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value -c uuid')
```

pc-m replied:
very nice, thanks for the info

sduthil commented on contribs/examples/enabled-video.sh:
```diff
@@ -0,0 +1,91 @@
+
+# Find WebRTC endpoints that do not have Video enabled
+echo -n 'Updating WebRTC endpoints'
+for endpoint_uuid in $(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value | awk -F ' ' '{ print $1 }'); do
```
```suggestion
for endpoint_uuid in $(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value -c uuid); do
```

pc-m committed:
example: print usage if argument are missing

pc-m committed:
enable video: protect against empty TENANT_UUID

pc-m committed:
enable video: simplify finding endpoint UUID

the `-c` option display only the specified columns removing the need for awk

pc-m committed:
enable video: protect against empty username

AUTH_USERNAME='' ./enable-video.sh

will not use the default user from wazo-auth-cli instead of prompting for a
password

pc-m committed:
rename script to enable video on webrtc

sduthil committed:
example enable-video: handle unknown arg

sduthil committed:
example enable-video: homogenize variable tests

sduthil committed:
example enable-video: shellcheck

sduthil committed:
example enable-video: refactor listing of endpoint sip
